pylzy/tests/api/v1/test_calls.py

```diff
@@ -368,7 +368,7 @@ def raises() -> int:
         self.assertEqual(int, func.output_types[0])
 
     def test_no_return_hint(self):
-        with self.assertRaisesRegex(TypeError, "return type is not annotated*"):
+        with self.assertRaisesRegex(TypeError, r"Return type is not annotated for function no_hint at .*"):
```

Comment:
let's wrap function name in quotation marks
i.e. `"Return type is not annotated for function 'no_hint' at .*"`

Reply:
I love using backticks in such scenarios for no reason
But I think single quotes will also be fine